features/support/route.js

```diff
@@ -163,6 +163,10 @@ module.exports = function () {
         return instructions.legs.map(l => {return zip(l.annotation.duration, l.annotation.distance).map(p => { return p.join(':'); }).join(','); }).join(',');
     };
 
+    this.geometryList = (instructions) => {
+        return instructions.geometry.coordinates.map(p =>{ return p[0] + ',' + p[1]; }).join(';');
```

Comment:
Relying on setting `geometries=geojson` here is bad. Easy to be messed up by the calling code. Instead I would suggest decoding the default (polyline) using https://github.com/mapbox/polyline

Also the parameter name `instructions` is misleading: it is a `Route` object.